misc/formatter/modules/formatter-core/src/main/java/org/ballerinalang/formatter/core/FormattingTreeModifier.java

```diff
@@ -913,10 +972,17 @@ public MetadataNode transform(MetadataNode metadataNode) {
 
     @Override
     public BlockStatementNode transform(BlockStatementNode blockStatementNode) {
-        if (!isInLineRange(blockStatementNode)) {
+        if (!isInLineRange(blockStatementNode, lineRange)) {
             return blockStatementNode;
         }
-        int startColumn = getStartColumn(blockStatementNode, blockStatementNode.kind(), false);
+        boolean addSpaces = false;
+        if (blockStatementNode.parent().kind().equals(SyntaxKind.NAMED_WORKER_DECLARATION) ||
+                blockStatementNode.parent().kind().equals(SyntaxKind.QUERY_ACTION) ||
+                blockStatementNode.parent().kind().equals(SyntaxKind.FOREACH_STATEMENT) ||
+                blockStatementNode.parent().kind().equals(SyntaxKind.FUNCTION_BODY_BLOCK)) {
```

Comment:
extract ` blockStatementNode.parent().kind()` to a variable

Reply:
Fixed.